src/ja/user/message/add.py

```diff
@@ -37,7 +37,7 @@ def execute(self, database: ServerDatabase) -> Response:
         if job.owner_id == -1:
             job.owner_id = self.effective_user
 
-        if self.effective_user != 0 and self.effective_user != job.owner_id:
+        if not self.effective_user_is_root and self.effective_user != job.owner_id:
```

Comment:
I see this as just as bad a security hole as before. I can run jobs as any user and get all of their files. After all, we want to protect the user's files.

Didn't we say we will remove submitting jobs as other users completely?

Reply:
Ah I am stupid, I didn't read it carefully enough. It is a good solution I think.